terraform_docs.sh

```diff
@@ -125,18 +118,18 @@ terraform_docs() {
 
     if [[ "$terraform_docs_awk_file" == "0" ]]; then
       # shellcheck disable=SC2086
-      terraform-docs md $args ./ > "$tmp_file"
+      terraform-docs md "${ARGS[*]}" ./ > "$tmp_file"
```

Comment:
`ARGS` needs to be unquoted to support multiple arguments like in #98 , otherwise `terraform-docs` errors out.
```suggestion
      terraform-docs md ${ARGS[*]} ./ > "$tmp_file"
```

Reply:
OK